templates/compose/appflowy.yaml

```
@@ -0,0 +1,217 @@
+      - APPFLOWY_MAILER_SMTP_EMAIL=${APPFLOWY_MAILER_SMTP_EMAIL:-notify@appflowy.io}
+      - APPFLOWY_MAILER_SMTP_PASSWORD=${APPFLOWY_MAILER_SMTP_PASSWORD:-email_sender_password}
+      - APPFLOWY_MAILER_SMTP_TLS_KIND=${APPFLOWY_MAILER_SMTP_TLS_KIND:-none}
+      - APPFLOWY_ACCESS_CONTROL=true
+      - APPFLOWY_DATABASE_MAX_CONNECTIONS=40
+      - AI_SERVER_HOST=ai
+      - AI_SERVER_PORT=5001
+      - AI_OPENAI_API_KEY=${AI_OPENAI_API_KEY:-}
+      - APPFLOWY_WEB_URL=${SERVICE_FQDN_APPFLOWYWEB_80}
+    depends_on:
+      gotrue:
+        condition: service_healthy
+      postgres:
+        condition: service_healthy
+      redis:
+        condition: service_started
+
+  gotrue:
+    image: appflowyinc/gotrue:latest
+    environment:
+      - SERVICE_FQDN_GOTRUE_9999
+      - GOTRUE_API_HOST=0.0.0.0
+      - GOTRUE_API_PORT=9999
+      - PORT=9999
+      - GOTRUE_ADMIN_EMAIL=${GOTRUE_ADMIN_EMAIL:-admin@example.com}
+      - GOTRUE_ADMIN_PASSWORD=${SERVICE_PASSWORD_ADMIN}
+      - GOTRUE_DISABLE_SIGNUP=false
+      - GOTRUE_SITE_URL=appflowy-flutter://
+      - GOTRUE_URI_ALLOW_LIST=**
+      - GOTRUE_JWT_SECRET=${SERVICE_PASSWORD_JWT}
+      - GOTRUE_JWT_EXP=${GOTRUE_JWT_EXP:-7200}
+      - GOTRUE_JWT_ADMIN_GROUP_NAME=supabase_admin
+      - GOTRUE_DB_DRIVER=postgres
+      - API_EXTERNAL_URL=${SERVICE_FQDN_GOTRUE_9999}
+      - DATABASE_URL=postgres://${SERVICE_USER_POSTGRES}:${SERVICE_PASSWORD_POSTGRES}@postgres:5432/${POSTGRES_DB:-appflowy}?search_path=auth
+      - GOTRUE_SMTP_HOST=${GOTRUE_SMTP_HOST}
+      - GOTRUE_SMTP_PORT=${GOTRUE_SMTP_PORT:-587}
+      - GOTRUE_SMTP_USER=${GOTRUE_SMTP_USER}
+      - GOTRUE_SMTP_PASS=${GOTRUE_SMTP_PASS}
+      - GOTRUE_MAILER_URLPATHS_CONFIRMATION=/verify
+      - GOTRUE_MAILER_URLPATHS_INVITE=/verify
+      - GOTRUE_MAILER_URLPATHS_RECOVERY=/verify
+      - GOTRUE_MAILER_URLPATHS_EMAIL_CHANGE=/verify
+      - GOTRUE_SMTP_ADMIN_EMAIL=${GOTRUE_SMTP_ADMIN_EMAIL}
+      - GOTRUE_SMTP_MAX_FREQUENCY=1ns
+      - GOTRUE_RATE_LIMIT_EMAIL_SENT=100
+      - GOTRUE_MAILER_AUTOCONFIRM=true
+    depends_on:
+      postgres:
+        condition: service_healthy
+    healthcheck:
+      test: "curl --fail http://127.0.0.1:9999/health || exit 1"
+      interval: 5s
+      timeout: 5s
+      retries: 12
+
+  admin_frontend:
+    image: appflowyinc/admin_frontend:latest
+    environment:
+      - SERVICE_FQDN_ADMINFRONTEND_3000
+      - RUST_LOG=info
+      - ADMIN_FRONTEND_REDIS_URL=redis://redis:6379
+      - ADMIN_FRONTEND_GOTRUE_URL=http://gotrue:9999
+      - ADMIN_FRONTEND_APPFLOWY_CLOUD_URL=http://appflowy_cloud:8000
+      - ADMIN_FRONTEND_PATH_PREFIX=/console
+      - ADMIN_FRONTEND_HOST=0.0.0.0
+      - ADMIN_FRONTEND_PORT=3000
+      - ADMIN_FRONTEND_OAUTH_CLIENT_ID=appflowy_cloud
+      - ADMIN_FRONTEND_OAUTH_CLIENT_SECRET=${SERVICE_PASSWORD_ADMINOAUTH}
+      - ADMIN_FRONTEND_OAUTH_ALLOWABLE_REDIRECT_URIS=${SERVICE_FQDN_ADMINFRONTEND_3000}
```

Since the latest update a few days ago, all `SERVICE_FQDN` and `SERVICE_URL` now work in reverse and need to be swapped.
```suggestion
      - ADMIN_FRONTEND_OAUTH_ALLOWABLE_REDIRECT_URIS=${SERVICE_URL_ADMINFRONTEND_3000}
```

---

fixed